app/controllers/logs_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LogsController < ApplicationController
+  def index
+    result = UseCases::Administrator::GetAuthRequestsForUsername.new(
```

Comment:
Intermediate `result` variable provide any value, over just assigning to `@logs` ?